shared/README.md

```diff
@@ -0,0 +1,17 @@
+# CodeQL Shared Libraries
+
+This folder contains shared, language-agnostic CodeQL libraries.
+
+Libraries are organized into separate query packs, in order to allow for
+individual versioning. For example, the shared static single assignment (SSA)
+library exists in the `codeql/shared-ssa` pack, which can be referenced by adding
+
+```
+dependencies:
+  codeql/shared-ssa: 0.0.1-dev
+```
+
+to `qlpack.yml`.
+
+All shared libraries will belong to a `codeql/shared-<name>` pack, and live in the
```

Comment:
I'm not convinced about putting the word `shared` all over the place. The whole point of packs and libraries is to share things, so there should be no need to state that in the name of a pack or package.

Why not simply use  `import codeql.<name>` for importing language-agnostic things? Language specific stuff can go into `codeql.<lang>` (or `semmle.code.<lang>`).

Reply:
Works for me.